Mergin/plugin.py

```diff
@@ -353,14 +353,15 @@ def view_local_changes(self):
             iface.messageBar().pushMessage("Mergin", "Current project is not a Mergin project.", Qgis.Warning)
             return
 
-        if not unsaved_project_check(force_saving=True):
+        proceed, reply_btn = unsaved_project_check()
+        if not proceed:
+            return
+        if reply_btn and reply_btn != QMessageBox.Yes:
```

Comment:
I think better to show this message in the diff viewer dialog. In current implementation messagebar partialy hidden and not very readable

Reply:
Done.